### ISSUE TITLE
volume control on mobile 

### ISSUE THREAD

**Issue by mishav78** [ableplayer/ableplayer#486]
Do all players have the issue of not being able to adjust volume on a mobile device?

**Comment by terrill** [comment]
@mishav78 No, mobile devices [now support controlling volume](https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/volume) through the HTML5 media API.  This needs to be revisited in Able Player.

**Comment by mishav78** [comment]
thanks
